source/apps/copy.txt

```diff
@@ -0,0 +1,200 @@
+.. _copy-app:
+
+===========
+Copy an App
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can make a copy of an existing App by reusing the App's
+configuration files and manually porting other information like Secrets.
```

Comment:
here it says "and manually porting other information like Secrets". what other info besides secrets would you have to manually port?

Reply:
I was thinking users when I wrote this but there's not really a way to port users so I'll update here